src/components/Proposal/Proposal.jsx

```diff
@@ -397,7 +399,10 @@ const Proposal = React.memo(function Proposal({
                     />
                   )}
                   {isVoteActive && (
-                    <>
+                    <Tooltip
+                      placement="bottom"
+                      content={`Block heights: ${startblockheight} - ${endblockheight}`}
```

Comment:
Let's update this to just:

```suggestion
                      content={`Block ${startblockheight} to ${endblockheight}`}
```